src/ppx/grfx/vk/vk_render_pass.cpp

```diff
@@ -133,6 +134,16 @@ Result RenderPass::CreateRenderPass(const grfx::internal::RenderPassCreateInfo*
     vkci.dependencyCount        = 1;
     vkci.pDependencies          = &subpassDependencies;
 
+    if (hasMultiView && pCreateInfo->multiViewMask > 0) {
+        VkRenderPassMultiviewCreateInfo multiviewInfo = {VK_STRUCTURE_TYPE_RENDER_PASS_MULTIVIEW_CREATE_INFO};
+        multiviewInfo.subpassCount                    = 1;
+        multiviewInfo.pViewMasks                      = &pCreateInfo->multiViewMask;
+        multiviewInfo.correlationMaskCount            = 1;
+        multiviewInfo.pCorrelationMasks               = &pCreateInfo->multiCorrelationMask;
+
+        vkci.pNext = &multiviewInfo;
```

Comment:
This is pointing to a variable that will be out of scope when used. `multiviewInfo` should be outside of the if statement.